src/mod/features/ev_iv_ui.cpp

```diff
@@ -0,0 +1,297 @@
+#include "exlaunch.hpp"
+#include "externals/il2cpp-api.h"
+
+#include "externals/Pml/PokePara/CoreParam.h"
+#include "externals/Dpr/Message/MessageWordSetHelper.h"
+#include "externals/System/Array.h"
+#include "externals/Dpr/UI/BoxStatusPanelUI.h"
+#include "externals/Dpr/UI/StatusPanelUI.h"
+#include "externals/Dpr/UI/StatusPanelLocals.h"
+#include "externals/System/String.h"
+
+
+
+int32_t POWER_ID_HP = 0;
+int32_t POWER_ID_ATK = 1;
+int32_t POWER_ID_DEF = 2;
+int32_t POWER_ID_SPATK = 3;
+int32_t POWER_ID_SPDEF = 4;
+int32_t POWER_ID_AGI = 5;
+
+extern void * int_TypeInfo;
+
+struct colorFields {
+    float r;
+    float g;
+    float b;
+    float a = 1.0f;
+};
+
+colorFields customColorFields;
```

Comment:
Instead of making this static, make mapColorToPower return it. Also it could be worth thinking about moving that function to our romdata folder. That folder will eventually have all files in it made into JSONs or similar in RomFS for easier tweaking